cypress/integration/unit_tests/upgrade.spec.ts

```diff
@@ -19,6 +20,31 @@ describe('Upgrade tests', () => {
 
   });
 
+  it('Create an OS Version Channels', () => {
+    // Create ManagedOSVersionChannel resource
+    cy.exec(`sed -i 's/# namespace: fleet-default/namespace: fleet-default/g' tests/assets/managedOSVersionChannel.yaml`)
```

Comment:
I dont know if `tests/assets/managedOSVersionChannel.yaml` could be in a variable? As it is wrote 2 times and can change. But that could be done later.